astarte/device/device.py

```diff
@@ -714,7 +714,7 @@ def __on_message(self, _client, _userdata, msg):
         if msg.payload:
             payload_object = bson.loads(msg.payload)
             data_payload = payload_object.get("v")
-            if not data_payload:
+            if data_payload is None:
```

Comment:
Maybe we could check if `"v" in payload_object` so we can handle when the property is null? Probably, it is overzealous.

Reply:
Yep, I think you are correct

Reply:
Thinking about it even properties should not be null. They would not reach this comparison as `msg.payload` would be empty for them.

Reply:
In any case, I think with your suggestion the code looks more readable. So I think it's worth the small overhead of iterating through the keys twice.

Reply:
Sorry, I maybe misspoke before. Mine was an additional check to the if, so we can log the missing v and the None separately. Because otherwise, we would not check if the property is resettable.